zerver/lib/push_notifications.py

```diff
@@ -382,18 +417,19 @@ def get_apns_payload(message):
 
 def get_gcm_payload(user_profile, message):
     # type: (UserProfile, Message) -> Dict[str, Any]
-    content = message.content
-    content_truncated = (len(content) > 200)
-    if content_truncated:
-        content = content[:200] + "..."
+    text_content = get_text_content(message.rendered_content or "")
+    content_truncated = False
+    if len(text_content) > 200:
+        text_content = text_content[:200] + "..."
+        content_truncated = True
```

Comment:
I think we should make a function, `truncate_content()`, that we use in both the GCM and APNS payloads.  

My thinking is if we do eventually include tags (and maybe even to handle unicode combining characters), if character 200 is in the middle of a tag or is a unicode combining character, we'll want to cut in a slightly different spot.

Also might want to replace the `...` with the unicode `...` symbol, and have the function cut it to <=200 characters, not <=203.